core/app/models/spree/image/active_storage_attachment.rb

```diff
@@ -3,10 +3,13 @@
 module Spree::Image::ActiveStorageAttachment
   extend ActiveSupport::Concern
   include Spree::ActiveStorageAdapter
-
```

Comment:
Why removing this line?

Reply:
There was no reasoning for this, it was overlooked and has been returned to the original state.